util/ranger/points.go

```diff
@@ -128,6 +128,12 @@ func FullIntRange() []IntColumnRange {
 	return []IntColumnRange{{LowVal: math.MinInt64, HighVal: math.MaxInt64}}
 }
 
+// FullIntNewRange is used for table range. Since table range cannot accept MaxValueDatum as the max value.
+// So we need to set it to MaxInt64.
+func FullIntNewRange() []*NewRange {
+	return []*NewRange{{LowVal: []types.Datum{types.NewIntDatum(math.MinInt64)}, HighVal: []types.Datum{types.NewIntDatum(math.MaxInt64)}}}
```

Comment:
It's better set exculde flag of highVal be true, which can avoid a `prefixNext`

Reply:
We cannot set it since this will not contain `MaxInt64`.